### PR TITLE
Update theme documentation

### PR DESCRIPTION
**Pull request by kgroschoff**
Updated the theme documentation in all versions

### CODE REVIEW

File: content/kubermatic/master/advanced/custom_ui/_index.en.md [hunk 1]
```diff
@@ -5,43 +5,73 @@ weight = 2
...
+  * [Customizing the Application Sources]({{< ref "#customizing-the-application-sources" >}})
+  * [Customizing the Application Sources Inside Custom Container]({{< ref "#customizing-the-application-sources-inside-custom-container" >}})
+  * [Customizing the Application Without Changing the Sources]({{< ref "#customizing-the-application-without-changing-the-sources" >}})
 
-All the application-wide rules are stored inside `src/assets/css` directory, where `_main.scss` is the main file that
-imports all other files that are required. The recommended approach to override application default styling is to
-register custom CSS files inside `_main.scss` and override existing CSS files only if it is needed. Overriding existing
-files is required when for example color variables need to be changed application-wide.
+## Modifying Available Themes
+Currently the application has two themes by default: light and dark. Each user can specify the theme that he want to use in the `User Settings` view which is accessible from the user menu.
```
Comment (floreks) [inline content/kubermatic/master/advanced/custom_ui/_index.en.md:20]: Do we need the screenshot of the UI here to show a theme switcher?

 > ... Each user can specify the theme that he want to use ...

s/want/wants

Maybe also change to "he/she" or simply change whole sentence to "Every user can select the theme in the `User Settings`..."

**Comment by maciaszczykm** [comment]
Will we be able to remove the manual from dashboard repo?

**Comment by kgroschoff** [comment]
@maciaszczykm Yes, I think so.

File: content/kubermatic/master/advanced/custom_ui/_index.en.md [hunk 2]
```diff
@@ -17,7 +17,9 @@ Here are some quick links to the different chapters:
   * [Customizing the Application Without Changing the Sources]({{< ref "#customizing-the-application-without-changing-the-sources" >}})
 
 ## Modifying Available Themes
-Currently the application has two themes by default: light and dark. Each user can specify the theme that he want to use in the `User Settings` view which is accessible from the user menu.
+Currently the application has two themes by default: light and dark. Each user can specify the prefered theme in the `User Settings` view which is accessible from the user menu.
```
Comment (maciaszczykm) [inline content/kubermatic/master/advanced/custom_ui/_index.en.md:20]: *preferred

**Comment by maciaszczykm** [comment]
Great :)

**Pull request merged**